wordpress-mobile/gutenberg-mobile
# [Embed block] Update bottom sheet to show specific embed label

Fixes https://github.com/wordpress-mobile/gutenberg-mobile/issues/3279

To test:
See Gutenberg PR https://github.com/WordPress/gutenberg/pull/34844

PR submission checklist:
